modules/nlu/src/backend/api.ts

```diff
@@ -51,27 +50,36 @@ export default async (bp: typeof sdk, state: NLUState) => {
       return res.status(404).send(`Bot ${botId} doesn't exist`)
     }
 
-    const modelId = botNLU.modelsByLang[botNLU.defaultLanguage]
+    const predictionLanguage = botNLU.defaultLanguage
```

Comment:
Is this route /predict still used somewhere ? 

I imagine only nlu-testing ?  speaking of nlu-testing... We'll soon be ready to ditch that whole module don't you think ?

Reply:
Yeah, we will ditch the module once we get rid of le legacy election pipeline. For now, this module is the only thing that ensures there's no regression outside of `nlu-core`.